installing/installing_oci/installing-c3-assisted-installer.adoc

```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="installing-c3-assisted-installer"]
+= Installing a cluster on {oci-c3-no-rt} by using the {ai-full}
+:context: installing-c3-assisted-installer
+
+toc::[]
+
+{oci-c3} (C3) lets you run applications and middleware using {oci-first} services on high performance cloud infrastructure in your data center. You can quickly deploy applications using a rack-scale platform with optional GPU acceleration and infrastructure-as-code automation. 
+
+// Overview
+include::modules/c3-assisted-installer-overview.adoc[leveloffset=+1]
+
+// Preinstallation considerations
+include::modules/c3-assisted-installer-preinstallation-considerations.adoc[leveloffset=+1]
+
+// Preparing the Bastian server
+include::modules/c3-assisted-installer-preparing-bastion-server.adoc[leveloffset=+1]
+
+// Terraform script execution - part 1
+include::modules/c3-assisted-installer-running-script-via-home.adoc[leveloffset=+1]
```

Comment:
Not sure  , there are two documents 1 for asisted-service and another for agent base installer.
To be on the safe side . the links here should be for assisted-installer ?

https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_assisted_installer.pdf?source=:em:nl:mt::::PCATP

Reply:
The link you added is for the CPA updates in Assisted Installer (next story).
The link to the C3 updates is: https://app.slack.com/client/E030G10V24F